srcpkgs/consolation/files/consolation/run

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+[ -r conf ] && . ./conf
+exec consolation $CONSOLATION_ARGS --no-daemon
```

Comment:
We generally don't ship the `conf` files for services and use `OPTS` as a generic variable name for passing arguments to services.